test/feature/traveler_registration_test.js

```diff
@@ -1,79 +1,79 @@
-import Nightmare from 'nightmare';
-import {expect} from 'chai';
-import {cleanDatabase, rootURL} from './feature_helpers';
+// import Nightmare from 'nightmare';
```

Comment:
We can't just comment out this test. Even if nightmare is indeed a nightmare.

Reply:
Then I think we we should just axe it entirely. Feature tests are too darn inconsistent to be of any real use. I think we instead opt to focus on testing more practical things like the redux actions.